hw/ip/rstmgr/data/rstmgr_testplan.hjson

```diff
@@ -30,17 +30,183 @@
       tests: ["rstmgr_smoke"]
     }
     {
-      name: feature1
-      desc: '''Add more test entries here like above.'''
-      milestone: V1
+      name: reset_stretcher
+      desc: '''Test the reset signal is stretched when stable.
+
+            The reset signal is stretched by 32 cycles when reset is stable.
+
+            **Stimulus**:
+            - Send a reset active signal, and de-activate it at random with
+              less than 32 cycles between de-activations for N de-activations.
+
+            **Checks**:
+            - The output reset is only set if the input reset has had at least
+              32 cycles of steady input reset active.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: sw_rst_regen_clear_once
```

Comment:
IMO we should move this to `hw/dv/tools/dvsim/testplans/csr_testplan.hjson` so that it is applied to all DUTs. @weicaiyang, @cindychip WDYT?

Reply:
Also, could we add a coverpoint (maybe inside `dv_base_reg_field` itself) to prove this is really covered by the CSR tests, for ALL regwens?

Reply:
looks like this isn't a regwen. Maybe it's due to regtool limitation since it contains multiple `regen` in a CSR and regwen usually only has one field. Perhaps @tjaychen can comment why we don't use regwen here.

For regwen, we relies on regular CSR tests and yes, we should consider to add covergroup for it.

Reply:
This regen enables per-bit "locking" of each peripheral's rst_n. The idea is that software can disable which peripherals can be reset via CSR, and that setting will persist until POR.

Reply:
Aah, I suppose I mistook this for regwen. My bad. 

However, my point still remains valid - we should add a testpoint for all the regwens since it relies on the CSR tests. So maybe create a separate tesplan with the testpoint and covergroup that IPs containing regwens can import.